createtemplates.sh

```diff
@@ -0,0 +1,327 @@
+#!/bin/bash
+
+# console url
+BASE_URL="https://console.stage.rafay.dev"
+API_KEY="<APIKEY>"
```

Comment:
As the utility is purely dependent on the correctness of. the APIKey being given , would'nt it be better to get a confirmation from user who is running this, saying here is the org that templates will be boostrapped and on confirmation continue ?